docs/docs/contributing-guide/marketplace/marketplace-setup.md

```diff
@@ -0,0 +1,51 @@
+---
+id: marketplace-setup
+title:  'Marketplace: Setup'
+---
+
+The Marketplace offers custom plugins that can be installed in your ToolJet workspace. This guide aims to assist you in creating a new plugin for the ToolJet marketplace.
+
+## Requirements
+- [Node.js](https://nodejs.org/en/download/) **(v18.3.0)**
+- [npm](https://www.npmjs.com/get-npm) **(v8.11.0)**
+
+## Getting started
+
+### Step 1. Enabling the marketplace for your instance
+
+In order to **activate** the marketplace for your ToolJet instance, you must set the following evironment variables in your **`.env`** file:
+
+#### Marketplace feature enable
+
+Use this environment variable to enable/disable the feature that allows users to use the [marketplace](/docs/marketplace).
+
+| variable                   | value             |
+| -------------------------- | ----------------- |
+| ENABLE_MARKETPLACE_FEATURE | `true` or `false` |
+
+#### Enable Marketplace plugin developement mode
+
+Use this environment variable to enable/disable the developement mode that allows developers to build the plugin.
```

Comment:
Can re-write to something similar, 'This environment variable enables developers to build plugins with greater ease by allowing for auto builds when there are changes in the packages. Additionally, a reload button is included for installed plugins to retrieve all the latest local changes from the file system. This feature is particularly useful for enhancing the development experience.'